.travis.yml

```diff
@@ -6,38 +6,20 @@ services:
   - docker
 env:
   global:
-    - AWS_DEFAULT_REGION: us-west-2
     - BOTO_CONFIG: "/dev/null"
     - AWS_XRAY_SDK_ENABLED: false
-    - USER_TABLE: testing
-    - DEPLOY_ENV: dev
+    - DEPLOYED_ENV: travis
     - secure: ldfVKFlO/kQLAmXFTf+UIsj1OzGV+EtfF1mEqp9ArD9eZg0EiM6mr2LTWMAKUyUlD3uCXSkF0xmIUIxNSUB+FwDxSNZdtO+Fx3hmF6I6ofX2NGEDdz2l/F7LeC3MfyPU7qwL9IL15gX2w6got78eDtdGMlhmt2s8dNsWDbLlAMjGZtCvfcXjY5GJS96Pe4lNHbCNlAi+x8E2UsyOBAEAySXfCfbS1JLW9XqiBKk1hinE+f2j47Wb6DJVX3JKfYLgdRdO+4JVa5dJvBIGhmO3lTi+vwDpTx+QbI+/oaoLpqSBpPwcpycZSPCJ0Oo+HfR8DCQRmT8NB6uc/wD44hrYlOXfXudaUH1TzQ2eT+ZT6BY5ikCArHpRDFHzlu2XmbTRq+phzArlV54H8xd46sIrMkLlVP4UHQ1GVOnXWTlG6wkrXxyPUFPAFeSKOiiK2UoHgz6j7yOTwKkuYH3/96GzH+YmoNZPFaDCh31YDFW4+ynlHFWA96U9+GKVS66f52aNMf7El6SwYSeuOzfOnKo5h5opKkkjO83KVrfzwBwpYkXHtyMG2o8hhJUDBF6VozPusJllkZfdyapdQgdTtH5R4Es1QNkzLAfX/iNGfr1yMc+I6O7VLEoB30202m1GDaocqnUwCHnMhnPdkw8E1x0DXoqSBHlCr/GJMjNiRsPjZ0k=
     - secure: eIOTi2ri6Y07/lqulILb6wQDC3SiZkiEPALG22icnOMQJJc7iSQCatyxZAze4M1aZRoQ3RgvKcftLFFB55JtXKwAoeQMU2eLSls31weKU4WgBqDwItJaKbbeDM/tBUYKel1rocx1rjxAGB/l4/YavlLwVyVOuA3MZ/bRbBEyBWsTJsdSpSUmEjXmWEcrk4ielMxZYBjbvSpa++FDDQvxSPzt5xeu66fa885i1u95MSNuaCJa3zkcueopEu1k/PA/IpqBr4Ga5llKer3epnByyy35JOnfBC4GC/Pu4B/520NaAj1YFP2Fuz9ZifSm9+E3iwHV1Ui87Xft8c3Qu0nIev9n9s3VUIq40Q5nWTmL9lolULdnQdWW5spHyKZGZKO4lX66KNe7Pj/B+58FUNYTzNkxhM1K1361QDrF9isgNgE/yCIcM4jeDHuZcnw0yjJe5b5Rv/RMIaEL/tz7DUSFKIWxbS6ESBVSyBoyxcph9b0mWH1LIKTWtFbFMS35Fxt962ozPmK/Wq+SXOfdPk7Um4Jz3o3/T12hFfcaqwXkeMtqRioq+m67+tYdWRUcflVyrXP4c3z5obLlo3t3bWRKSrr3ca5j60yxt+wbD+jt53A4hR9jrlRXsiMlm4T0GOE10loF2rYIzrnL+2yOD8cpaKG0StqLh4IUYeJk4X7WtlY=
```

Comment:
These 2 credentials are no longer utilized and can be removed.

Reply:
ok removing